solutions/Javascript_Variables_and_Strings/02_Hello_Name/helloName.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title></title>
+    </head>
+    <body>
+        <script src="helloName.js"></script>
```

Comment:
I'd suggest putting your script tags in the `head` section of the html document (only in rare cases it'd be wiser to put in the `body`, but not this one -- ask me for more info if you're curious :wink: )